src/providers/Providers.tsx

```diff
@@ -1,13 +1,16 @@
 import type { PropsWithChildren } from "react";
 import React from "react";
+import { I18nProvider } from "react-aria-components";
 
 import { ReactQueryProvider } from "./ReactQueryProvider";
 import { SentryProvider } from "./sentry/SentryProvider";
 
 export const Providers = ({ children }: PropsWithChildren) => {
   return (
     <ReactQueryProvider>
-      <SentryProvider>{children}</SentryProvider>
+      <I18nProvider locale='en-US'>
```

Comment:
By default `react-aria-components` uses the client locale for calendar labels. If we need to override them to english or any other language we have to use the provider.